testing/bdd/features/timing/delay.feature

```diff
@@ -20,29 +20,29 @@ Feature: Delay of a document sequence
     # Negative value should really throw an exception
     | 00:00:10.0 | -00:00:00  | 00:00:10.0         |  
 
-#TODO: test that the order of documents in the input and output sequence is the same (FIFO) 
+#TODO: test that the order of documents in the input and output sequence is the same (FIFO)
 
 
   #TBC: this is a change to the spec (passive delay node)
   @skip
```

Comment:
I have the impression that this PR resolves #293 also, so we should unskip them and check they pass, i.e. implement in this PR if possible.

Yes, remove the line about negative delay.